client/src/components/TopBar.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+
+import DragHandleIcon from '@material-ui/icons/DragHandle';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import clsx from 'clsx';
+import { Link } from "react-router-dom";
+
+import UserContext from '../context/User'
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		flexGrow: 1,
+	},
+	menuButton: {
+		marginRight: theme.spacing(2),
+	},
+	title: {
+		flexGrow: 1,
+	},
+	AppBar: {
+		backgroundColor: '#fff',
+		color: '#000',
+	},
+}));
+
+
+
+export default function TopBar(props) {
+
+	const classes = useStyles();
+	const user = useContext(UserContext)
+	const [auth, setAuth] = useState(user.role);
+	const [anchorEl, setAnchorEl] = useState(null);
+	const open = Boolean(anchorEl);
+
+	const routes = ['chef']
+	const [top, setTop] = useState({
+		top: false,
+	});
+	const handleAuth = (event) => {
+		setAuth(event.target.checked);
+	};
```

Comment:
Are we just using this for debug right now?

Reply:
I'm currently using it as a toggle for the user icon in the nav bar which will be hidden when the user is logged out. I figured eventually this function would be used to trigger the logout in a future task.

Reply:
should remove the handleAuth function?

Reply:
handleAuth is removed